frontend/pages/organizations/index.vue

```diff
@@ -24,21 +24,15 @@
 </template>
 
 <script setup lang="ts">
-import { Organization } from "../../types/organization";
 
 definePageMeta({
   layout: "sidebar",
 });
-
-const organization: Organization = {
```

Comment:
I'm going to throw this in as a comment in case we want to check this without a backend connection. Will remove as soon as we have a default org that we can put where ever via fixtures :)